chemper/smirksify.py

```diff
@@ -40,14 +40,33 @@
 
 import networkx as nx
 import time
-from chemper.graphs.environment import ChemicalEnvironment
+from chemper.graphs.environment import ChemicalEnvironment as CE
 from chemper.mol_toolkits import mol_toolkit
 from chemper.chemper_utils import ImproperDict, ValenceDict, \
     get_typed_molecules, is_valid_smirks, match_reference
 
-import numpy
+import numpy as np
 from numpy import random
 
+#
+# private subroutines
+#
+def print_smirks(smirks_list):
+    """
+    Prints out the current or provided smirks list
+    in a table like format
+    """
+    str_form = " {0:<20} | {1:} "
+
+    print()
+    print(str_form.format("Label", "SMIRKS"))
+    print('='*80)
```

Comment:
What is "*80" for?

Reply:
This whole function is just to make printing SMIRKS prettier. 
`print('='*80)` 
leads to this line:
`================================================================================`

Here's an example of an output table:
```
Label                | SMIRKS 
================================================================================
 zz_sing              | [*:1]~[*:2] 
--------------------------------------------------------------------------------
 zz_aromatic          | [*:1]:[*:2] 
--------------------------------------------------------------------------------
 zz_double            | [*:1]=[*:2] 
--------------------------------------------------------------------------------
```

Its for human readability, as a table, not computer readability.

Reply:
👍